lib/buffer.js

```diff
@@ -1,13 +1,18 @@
+/* eslint-disable no-class-assign */
 /* eslint-disable require-buffer */
 'use strict';
 
 const binding = process.binding('buffer');
 const { isArrayBuffer } = process.binding('util');
+const { deprecate } = require('internal/util');
 const bindingObj = {};
 
-class FastBuffer extends Uint8Array {}
+class Buffer extends Uint8Array {}
+
+// to avoid code churn...
+const FastBuffer = Buffer;
+Buffer = DummyBuffer;
```

Comment:
Well, that would mean changing all the "static" function declarations, e.g. `Buffer.alloc` to `DummyBuffer.alloc`, only to change them back later when Buffer becomes a proper class.